lemur/utility.py

```diff
@@ -271,12 +281,18 @@ def observation_number_for_experiment(experiment_id):
 
 def serialize_lab_list(lab_list):
     labs = []
+
     for lab in lab_list:
+        if lab.the_class:
+            class_id = lab.the_class.id
+        else:
+            class_id = None
+        prof_name = ','.join([u.name for u in lab.users if u.role_name != 'Student'])
```

Comment:
this method may produce the wrong value if you add a role name in the future that does not refer to a professor and is not Student.